tt_metal/hw/inc/dev_msgs.h

```diff
@@ -91,7 +93,17 @@ struct launch_msg_t {  // must be cacheline aligned
     volatile uint8_t dispatch_core_x;
     volatile uint8_t dispatch_core_y;
     volatile uint8_t exit_erisc_kernel;
-    volatile uint8_t run;  // must be in last cacheline of this msg
+    volatile uint8_t pad1;
+    volatile uint16_t pad2;
```

Comment:
why do we need this 2 byte padding?

Reply:
We need to be 32-bit aligned on this struct, cache aligned.